SU-SWS/caw_profile
# CAW21-101 Start one time edit link for career celebrations

# READY FOR REVIEW

# Summary
- Add functionality for a one time edit link to be emailed to a person. This prevents them from needing access to anything else within the site. They only have access to edit the one node and no need to log into the site.

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. checkout branch
2. `blt drupal:update --site=cardinalatwork`
3. Create a "Career Celebrations" node and in the right, put in an email address for the one-time edit link
   - You might not get an email, depending on your server setup
4. after saving, you will see a url in the status message, copy that and paste into an icognito window
5. modify the fields in the icognito window
6. save and verify the url you used no longer works.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
